zsh-pyenv.plugin.zsh

```diff
@@ -17,6 +17,7 @@ _zsh_pyenv_install() {
 
 _zsh_pyenv_load() {
     eval "$(pyenv init - --path --no-rehash zsh)"
+    eval "$(pyenv init -)"
```

Comment:
The first argument dash is no longer necessary. Also `--no-rehash zsh` goes on the next line.
```suggestion
    eval "$(pyenv init --path)"
    eval "$(pyenv init - --no-rehash zsh)"
```

Reply:
Thanks @DanielPower. I was having trouble making the original PR work on my system, and was digging into the proper syntax. Your changes seem to make it work.